psql2mysql/__init__.py

```diff
@@ -377,8 +394,8 @@ def do_prechecks(config, source, target):
 
 def do_migration(config, source, target):
     migrator = DbDataMigrator(config, source, target)
-    migrator.setup()
```

Comment:
I'd still prefer there was some setup method at least for testing sake, makes it easier to separate it out into logical pieces.

Reply:
Hm, ok. But what would it do? I move the connect() out of it because e.g. the do_purge_tables does not need to connect to the source database. So if I reintroduce setup() the whole change to the srcdb/targetdb properties is moot. Or did you mean something else?

Reply:
I'm just a greybeard that doesn't like magical caching properties, get off my lawn. :-P